Source/WebKit/Resources/SandboxProfiles/ios/com.apple.WebKit.WebContent.sb.in

```diff
@@ -1404,6 +1411,7 @@
         mach_port_set_attributes
         mach_vm_copy
         mach_vm_map_external
+        (when (defined? 'mach_vm_range_create) mach_vm_range_create) ;; <rdar://105161083>
```

Comment:
I think also this sys call could only be allowed before the process has finished launching.